articles/protocols/oidc/index.md

```diff
@@ -5,47 +5,27 @@ description: What is the OpenID Connect protocol and how it works.
+## ID Tokens
 
-In addition to these standard claims, other claims may be used as well.
+The [ID token](/tokens/id_token) is a [JSON Web Token (JWT)](/jwt) that contains identity data. It is consumed by the client and used to get user information like the user's name, email, and so forth, typically used for UI display. ID Tokens conforms to an industry standard (IETF [RFC 7519](https://tools.ietf.org/html/rfc7519)) and contain three parts: a header, a body and a signature.
 
-For example, we could add an extra property at our client's users using [User Metadata](/metadata). This property would be called `roles` and it would have either the value `ROLE_ADMIN` or the value `ROLE_USER`. The plan is to get this value upon user login and depending on the value customize the UI so the user can or cannot take admin actions.
+### Claims
 
-All we have to do now is specify this new metadata as part of the `scope` request parameter (`scope=openid roles`). Once we decode our ID Token the body payload will contain the property `roles` with the corresponding value for the logged in user.
+ID Tokens contain claims, which are statements (such as name or email address) about an entity (typically, the user) and additional metadata. There are three [types of claims](/jwt#payload): reserved, public, and private claims. 
 
-## How the protocol works
+The [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html) defines a set of standard claims, but other claims (including self-defined, custom claims) may be used in addition to the standard claims.
 
-Let's use the example we mentioned earlier, signing into Auth0 using your Google account, for a high level overview on how the flow works:
+## Access Tokens
 
-1. When you choose to sign in to Auth0 using your Google account, Auth0 sends an **Authorization Request** to Google.
-1. Google authenticates your credentials or asks you to login if you are not already signed in, and asks for your authorization (lists all the permissions that Auth0 wants, for example read your email address, and asks you if you are ok with that).
-1. Once you authenticate and authorize the sign in, Google sends an ID Token, and usually an Access Token, back to Auth0.
-1. Auth0 can retrieve user information from the ID Token or use the Access Token to invoke a Google API.
+[Access tokens](/tokens/access-token) are credentials that can be used by a client to access an API (or other protected resource). Access tokens can be an opaque string, JWT, or non-JWT token. Its purpose is to inform the API that the bearer of this token has been authorized to access the API and perform specific actions (as specified by the scopes that have been granted).
```

Comment:
First sentence: what other protected resource except for APIs can access tokens be used to access?